horace_core/sqw/coord_transform/@projection/private/get_contributing_pix_ind_.m

```diff
@@ -20,5 +23,9 @@
 % get good indices (including integration axes and plot axes with only one bin)
 % roun-off them to 7 significant digit to eliminate round-off errors,
 % generated by epsilon around the urange border
-indx=round(indx(ok,:),7);
+
+indx  = indx(ok,:);
+%indx=round(indx(ok,:),7); %-- very slow. Below seems much faster. Consider
+%                          % fully removing this operation
+%indx=floor(10000000*indx(ok,:))/10000000; -- much faster then round
```

Comment:
but as it is all commented out this is just removing rounding. Checking that is what is intended.

Reply:
it is intended. Correspondent round-off is done in the beginning over ranges. There is something dodgy with the ranges in the code due to round-off errors. I will pay attention to it when refactoring to projection but keep the code commented as the reminder of the problem for the time being.